Keyboard/DivvunSpellBannerPlugin.swift

```diff
@@ -20,15 +20,17 @@ class SuggestionOp: Operation {
         guard let speller = plugin.speller else { return }
 
         
+        let currentWord = BannerItem(title: "\"\(word)\"", value: word)
+        
         let suggestions = (try? speller
             .suggest(word: word)//, count: 3, maxWeight: 4999.99)
-            .prefix(3)
-            .map { BannerItem(title: $0, value: "suggestion") }) ?? []
```

Comment:
I didn't see a benefit of using a `value` of `"suggestion"` here. I assume this was being used as some sort of tag, but couldn't find it being used anywhere else in the project. I've repurposed the `value` attribute to be the actual value to be used when replacing text. This way `title` and `value` can differ. Let me know if a value of `"suggestion"` was important.